src/OpenToolkit.Graphics/ES30/Helper.cs

```diff
@@ -24,10 +24,10 @@
 //
 
 using System;
-using System.Diagnostics;
 #if !MINIMAL
 using System.Drawing;
 #endif
+using System.Diagnostics;
```

Comment:
What was S.Diagnostics for? If not used, please remove.

Reply:
This is outside the scope of this PR

Reply:
Please elaborate